umap/static/umap/js/modules/data/layer.js

```diff
@@ -1071,7 +1065,13 @@ export class DataLayer {
               'This situation is tricky, you have to choose carefully which version is pertinent.'
           ),
           async () => {
+            // Save again this layer
             await this._trySave(url, {}, formData)
```

Comment:
What if there is still an error, shouldn't we check that `true` is returned here?

Reply:
Indeed! Good point :)

Reply:
Fixed!